index.js

```diff
@@ -72,13 +72,14 @@ function beautify(error, collection, values, messages) {
                 indexes[indexName].forEach(function (field) {
                     var path = field[0];
                     var props = {
-                        type: 'Duplicate value',
+                        type: 'unique',
+                        kind: 'unique',
                         path: path,
                         value: values[path]
                     };
 
                     if (typeof messages[path] === 'string') {
-                        props.message = messages[path];
+                        props.message = props.reason = messages[path];
```

Comment:
Looks like the `reason` attribute [is not being used by most validators](https://github.com/Automattic/mongoose/blob/4.11.1/lib/schema/string.js#L108), so do you think we should provide it? If so, I’m not sure whether it’s appropriate to set it to the same value as `message`.